.github/workflows/stalebot.yaml

```diff
@@ -0,0 +1,34 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          stale-issue-message: 'This issue has been automatically marked as stale due to lack of activity. You can remove the stale label or comment. Otherwise, this issue will be closed in 7 days. Thank you!'
+          close-issue-message: 'We are closing this issue due to lack of activity. Feel free to reopen it if you can provide more information. Thank you!'
+       
+          # Don't process PRs 
+          days-before-stale: -1
+
+          # Process only issues
+          days-before-issue-stale: 14
+          days-before-issue-close: 7
+          
+          # Add this label after 'exempt-issue-labels' days to mark it as stale
+          stale-issue-label: 'no-issue-activity'
+          
+          # Stale only issues with one of these labels
+          any-of-issue-labels: 'bug,enhancement'
+          
+          # Exclude issues with the 'in-progress' label
+          exempt-issue-labels: 'in-progress'
+          
```

Comment:
This label is something that people will forget to add or remove, so I wonder how useful it is. Is it possible to check if the bug is assigned to anybody instead of using a label?

Reply:
My idea was to add this label automatically as soon as we move the issue to the 'ready' column in our board (if this is possible in a GitHub workflow). However, I added a further filter. Assigned issues will also be excluded.